cookbooks/bcpc-hadoop/recipes/graphite_to_zabbix.rb

```diff
@@ -306,5 +309,5 @@
   minute "*"
   hour   "*"
   user   "nobody"
-  command  "/usr/local/bin/run_zabbix_sender.sh"
+  command  "pgrep 'zabbix_sender' > /dev/null || /usr/local/bin/run_zabbix_sender.sh"
```

Comment:
Do you want to direct `pgrep` to look only for processes run by this user or anything more specific to avoid unexpected interactions? We've hit issues in the past not being terribly specific with Chef services for example, as in Chef-BACH #53.